VAMobile/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -235,6 +280,12 @@ export function HomeScreen({}: HomeScreenProps) {
         <Box mb={theme.dimensions.contentMarginBottom}>
           <SimpleList items={buttonDataList} />
         </Box>
+        <Box mb={theme.dimensions.contentMarginBottom}>
+          <AnnouncementBanner
+            title={t('learnAboutPACT')}
```

Comment:
Since the title has the text `VA.gov` in it, you'll need to set the accessibility label to `a11yLabelVA(t('learnAboutPACT'))` because TalkBack/VoiceOver on certain devices won't pronounce `VA` correctly. You'll probably need to add an accessibility label prop to the component for setting the a11y label